commons/connector/health/health_activity.avsc

```diff
@@ -0,0 +1,24 @@
+{
+  "namespace": "org.radarcns.connector.health",
```

Comment:
If this is a HealthKit connector, please rename the namespace (and directory) to `org.radarcns.connector.healthkit`. If it is meant to be integrated into the aRMT app, please rename (and move) to `org.radarcns.active.healthkit`